version1/02 - Object types.md

```diff
@@ -404,49 +404,51 @@ The following notation is used in the tables below:
+| M603      | string    | avsluttetAvBrukerIdent           |          |             |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                                   |
+| M004      | string    | registreringsIdent               |    x     |      x      |                    |  |  cw  |  x   |   x   |  x   |      |        |                                                                                                                                                                   |
+| M020      | string    | tittel                           |    x     |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                   |
+| M025      | string    | offentligTittel                  |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                   |
+| M021      | string    | beskrivelse                      |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                   |
+| M024      | string    | forfatter                        |          |             |                    |  |  x   |  x   |   x   |  x   |      |        |                                                                                                                                                                   |
+|           | string    | publiseringsstatus               |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | Allowed values: NOT_PUBLISHED, PUBLISHED_METADATA, PUBLISHED_METADATA_AND_DOCUMENTS                                                                               |
+| M300      | string    | dokumentmedium                   |    x     |      x      | dokumentmedium     |  |  x   |  x   |   x   |      |      |        |                                                                                                                                                                   |
+|           | string    | journalansvarlig                 |    x     |      x      |                    |  |  x   |  x   |   x   |  x   |      |        | <p>&bull; The name of the person responsible for this Basisregistrering (similar to M306 Saksmappe#saksansvarlig)</p>                                             |
```

Comment:
Unlike `Journalpost` the fields can be null for the other types and they don't have a default value so the x-marks need to be removed.

Reply:
Great catch :) Fixed with https://github.com/documaster/noark5-web-services/pull/64/commits/d39d319323ac0605789129aad5035ebccd4054a9 

With the same commit I have added documentation for the same fields and for the Arkivnotat.